### ISSUE TITLE
Public IP Changing Doesn't Update

### ISSUE THREAD

**Issue by necromancyr** [Razzmatazzz/ValheimServerWarden#38]
This is going to be a difficult one to test around I think, but it looks like the external IP is checked on VSW start and then not updated/rechecked each time the server is updated.  This only impacts the Discord announcement and connection string feature, so it's not a major issue, but could be resolved with an external  IP check prior to server start (if it's not already).

**Comment by Razzmatazzz** [comment]
That is correct; VSW only checks the public IP when the app is started and then assumes that it doesn't change. It's my understanding that one's public IP isn't likely to change frequently, so I figured this shouldn't be much of an issue.
